Common/Setup.py

```diff
@@ -138,6 +140,8 @@ def __init__(self, ana_path, period, customisations=None):
             if type(sample_entry) != dict:
                 raise RuntimeError(f'Invalid sample definition period="{period}", sample_name="{sample_name}"')
         selected_samples = select_items(samples.keys(), self.global_params.get('sampleSelection', []))
+        if sample: # If you define a user cample, then only that sample is selected
+            selected_samples = [s for s in selected_samples if s == sample]
```

Comment:
use `select_items(selected_samples, [ 'drop *',  f'keep {sample}']) to allow regex in sample selection